### PR TITLE
[328] Standout list BE

### PR DESCRIPTION
**Pull request by richbrennan**
Implements https://projects.torchbox.com/projects/tbxcom/tickets/328

- Adds a new streamfield to the culture page for adding standout items (see attached screenshot)
- Renames culture page to careers page in wagtail admin
- Updates shared templates and associated yaml files
- Adds methods to the culture and service pages to format data for shared templates

Also adds PUDB to the build for delightful debugging.

Screenshot from 2021-03-25 10-10-18

### CODE REVIEW

File: tbx/people/models.py [hunk 1]
```diff
@@ -146,8 +177,44 @@ class CulturePage(Page):
...
+
+        return [
+            {
+                "title": x.value["title"],
+                "subtitle": x.value["subtitle"],
+                "description": x.value["description"],
+                "url": get_link(x.value["link"]),
+                "image": x.value["image"],
+            }
+            for x in self.standout_items
```
Comment (BrianXu20) [inline tbx/people/models.py:210]: for standout_item in self.standout_items. seems more readable

File: tbx/people/models.py [hunk 2]
```diff
@@ -128,6 +130,35 @@ class CulturePage(Page):
...
+                "item",
+                blocks.StructBlock(
+                    [
+                        ("subtitle", blocks.CharBlock()),
+                        ("title", blocks.CharBlock()),
+                        ("description", blocks.TextBlock()),
+                        ("image", ImageChooserBlock()),
+                        (
+                            "link",
+                            blocks.StreamBlock(
```
Comment (BrianXu20) [inline tbx/people/models.py:145]: Seems it's better to create a StreamBlock for link field  and put the def get_link(link_value) from line 189 there
Just like class LinkFields in the project

File: tbx/services/models.py [hunk 3]
```diff
@@ -148,6 +148,20 @@ def serve(self, request):
...
+        """Format the featured case studies data for the template."""
+        return [
+            {
+                "title": x.case_study.title,
+                "subtitle": x.case_study.client,
+                "description": x.case_study.listing_summary,
+                "url": x.case_study.url,
+                "image": x.case_study.homepage_image,
+            }
+            for x in self.featured_case_studies.all()
```
Comment (BrianXu20) [inline tbx/services/models.py:161]: for case_study in self.featured_case_studies.all() seems better

**Review by BrianXu20 — APPROVED** [review]
Looks good to me, no major issue found

File: tbx/services/models.py (hunk 3 above)
Comment (richbrennan) [reply tbx/services/models.py:161]: That would result in `casestudy.casestudy` in the list comprehension, but I agree `x` isn't the best. I'll go with  `f` for `featured_case_study`

**Comment by chris-lawton** [comment]
Looks good @richbrennan! I'm going to merge it to the FE branch and then pick up the remaining changes there. Thanks!

**Pull request merged**